reporter/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json

```diff
@@ -10,5 +10,107 @@
   "name" : "some document name",
   "dataLicense" : "CC0-1.0",
   "comment" : "some document comment",
-  "documentNamespace" : "<REPLACE_DOCUMENT_NAMESPACE>"
+  "documentNamespace" : "<REPLACE_DOCUMENT_NAMESPACE>",
+  "packages" : [ {
+    "SPDXID" : "SPDXRef-Package-0-root-package",
+    "copyrightText" : "NOASSERTION",
+    "downloadLocation" : "NOASSERTION",
+    "filesAnalyzed" : false,
+    "homepage" : "NOASSERTION",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "NOASSERTION",
+    "name" : "Root package",
+    "summary" : "An artificial stub package describing the root of the dependency tree."
+  }, {
+    "SPDXID" : "SPDXRef-Package-1-first-package",
+    "copyrightText" : "Copyright 2020 Some copyright holder\nCopyright 2020 Some other copyright holder",
+    "downloadLocation" : "https://some-host/first-package.jar",
+    "filesAnalyzed" : false,
+    "homepage" : "first package's hompage URL",
+    "licenseConcluded" : "NOASSERTION",
+    "licenseDeclared" : "BSD-3-Clause AND (MIT OR GPL-2.0-only)",
+    "name" : "Maven:first-package-group:first-package",
+    "summary" : "A package with all supported attributes set, with a VCS URL containing auser name, and with a scan result containing two copyright finding matched to a license finding.",
```

Comment:
Spelling "auser name" -> "an user name"

Why do you say "with a VCS URL containing auser name" as spec says "Using user names and password in the <host_name> is not supported and should be considered as an error." in https://spdx.github.io/spdx-spec/3-package-information/#37-package-download-location

Reply:
Fixed the type. I'm using a user name to illustrate that it get properly stripped.